src/transformers/pipelines/text_generation.py

```diff
@@ -285,6 +289,9 @@ def __call__(self, text_inputs, **kwargs):
                 - `None` : default strategy where nothing in particular happens
                 - `"hole"`: Truncates left of input, and leaves a gap wide enough to let generation happen (might
                   truncate a lot of the prompt and not suitable when generation exceed the model capacity)
+            tokenizer_kwargs (`dict`, *optional*):
```

Comment:
perhaps `tokenizer_encode_kwargs`? There are also kwargs used at decode time, and we don't want to mix the two